<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="chetan" ignored="0" name="Car loan" tests="7" failures="1" timestamp="13 Feb 2019 14:12:02 GMT" time="0.091" errors="0">
  <testcase name="apiCarLOan" time="0.01" classname="BasicTest.Day3"/>
  <testcase name="failMethod" time="0.007" classname="BasicTest.Day3">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at BasicTest.Day3.failMethod(Day3.java:27)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- failMethod -->
  <testcase name="mobCarLoan" time="0.002" classname="BasicTest.Day3"/>
  <testcase name="webCarLoan" time="0.002" classname="BasicTest.Day3"/>
  <testcase name="webHomeLoan" time="0.002" classname="BasicTest.Day4"/>
  <testcase name="mobHomeLoan" time="0.002" classname="BasicTest.Day4"/>
  <testcase name="apiHomeLOan" time="0.001" classname="BasicTest.Day4"/>
</testsuite> <!-- Car loan -->
